dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/validator/relationship/ValidationUtils.java

```diff
@@ -45,19 +45,19 @@ private ValidationUtils()
         throw new IllegalStateException( "Utility class" );
     }
 
-    public static TrackerType relationshipItemValueType( RelationshipItem item )
+    public static String relationshipItemValueType( RelationshipItem item )
```

Comment:
Not sure why this is needed.
I would rather use the Enum itself to do any future comparison than get the name here and then check if 2 strings are equal. You even needed to use `equalsIgnoreCase` in `DataRelationsValidator`